jax/_src/export/_export.py

```diff
@@ -372,19 +372,53 @@ def out_shardings_jax(
                  for s in self.out_shardings_hlo)
 
   def has_vjp(self) -> bool:
+    """Returns if this Exported supports VJP."""
     return self._get_vjp is not None
 
   def vjp(self) -> Exported:
     """Gets the exported VJP.
 
     Returns None if not available, which can happen if the Exported has been
-    loaded from an external format, without a VJP."""
+    loaded from an external format without a VJP.
+    """
     if self._get_vjp is None:
       raise ValueError("No VJP is available")
     return self._get_vjp(self)
 
+  def serialize(self,
+                vjp_order: int = 0) -> bytearray:
+    """Serializes an Exported.
+
+    Args:
+      vjp_order: The maximum vjp order to include. E.g., the value 2 means that we
+        serialize the primal functions and two orders of the `vjp` function. This
+        should allow 2nd order reverse mode differentiation of the deserialized
+        function. i.e., `jax.grad(jax.grad(f)).`
+    """
+    # Lazy load the serialization module, since flatbuffers is an optional
+    # dependency.
+    from jax._src.export.serialization import serialize
+    return serialize(self, vjp_order=vjp_order)
+
+  def call(self, *args, **kwargs):
```

Comment:
Why not implement it as `__ call__`?

Compile also has an `__call__` so why not follow the same route? And it's pretty standard for python callables to do this.

Reply:
This may be a matter of personal preference. I am thinking of the Exported primarily as a dataclass whose primary purpose is to be ready for serialization, and only secondarily as something that can be turned into a Callable. Hence, I prefer that this process be explicitly invoked. E.g., I found that writing `jax.jit(exp.call)` in the tests to be clearer than `jax.jit(exp)`. 

Also, it is easier to add later a `__call__` if we really think it is important rather than adding it now and then regretting it.

Reply:
Sounds good.